lib/errors/validation-error.ts

```diff
@@ -195,11 +205,13 @@ export class ValidationErrorItem {
 }
 
 /**
- * Validation Error. Thrown when the sequelize validation has failed. The error contains an `errors` property,
- * which is an array with 1 or more ValidationErrorItems, one for each validation that failed.
+ * Thrown when the sequelize validation has failed. The error contains an `errors` property,
+ * which is an array with 1 or more [ValidationErrorItem](/esdoc/class/lib/errors/validation-error.ts~ValidationErrorItem.html), one for each validation that failed.
```

Comment:
This was also written as markdown. Use `{@link ValidationErrorItem}` instead

Reply:
The @link tag doesn't output anything in docs build output.

Reply:
I just tried generating the doc using `{@link ValidationErrorItem}` and it worked

Reply:
> I just tried generating the doc using `{@link ValidationErrorItem}` and it worked

Mmmh noted, will try to investigate what happen this night.

Reply:
Sorry I was mistaken. I think I edited the built file by accident. Looking more into this

Reply:
Apparently esdoc hasn't had a commit in a couple of years. So it's unlikely this will get fixed.

I think we could consider moving to https://typedoc.org/ or similar?

Reply:
I did a quick test with typedoc and it worked pretty well :)

I propose we pause this PR until a migration to something that properly supports typescript for doc generation

<img width="841" alt="Screenshot 2022-01-07 at 10 15 49" src="https://user-images.githubusercontent.com/1280915/148521255-f3981855-9db2-495b-be71-c0936183cf81.png">

Reply:
> I did a quick test with typedoc and it worked pretty well :)
> 
> I propose we pause this PR until a migration to something that properly supports typescript for doc generation
> 
> <img alt="Screenshot 2022-01-07 at 10 15 49" width="841" src="https://user-images.githubusercontent.com/1280915/148521255-f3981855-9db2-495b-be71-c0936183cf81.png">

Ultimate agree! what do you think @sdepold ?

Reply:
I'll redirect this conversation to https://github.com/sequelize/sequelize/issues/13769 :)